plugin/omero_iviewer/views.py

```diff
@@ -219,8 +219,8 @@ def persist_rois(request, conn=None, **kwargs):
     return JsonResponse(ret)
 
 
-def get_query_for_rois_by_plane(the_z=None, the_t=None, z_end=None,
-                                t_end=None, load_shapes=False):
+def get_query_for_rois_by_plane(the_z=None, the_t=None,
+                                z_end=None, t_end=None):
```

Comment:
Does this count as a breaking change?

Reply:
This method hasn't been released yet. It was introduced in https://github.com/ome/omero-iviewer/pull/342 which was merged after the last (0.10.1) release.

Reply:
:+1: